scripts/facts.py

```diff
@@ -403,7 +403,7 @@ def handleCnamesOfHost(primaryName, cnameConfig):
     # Internal Records generation
     if 'dns_facts_internal_records' in localhost:
         if 'subdomain' in localhost['dns_facts_internal_records']:
-            subdomain = localhost['dns_facts_internal_records']['subdomain']
+            subdomain = localhost['dns_facts_internal_records']['subdomain'] + '.'
```

Comment:
Doesn't this mean (with your docs above) that the dot is doubled now? Change looks good by itself, but this one line seems incorrect

Reply:
The doc above is for a different function.

Reply:
Oh well